src/PointwiseFunctions/Hydro/Tags.hpp

```diff
@@ -211,4 +213,17 @@ struct MassFlux : db::SimpleTag {
   static std::string name() { return Frame::prefix<Fr>() + "MassFlux"; }
 };
 }  // namespace Tags
+
+/// %Tags for options of hydrodynamic systems.
+namespace OptionTags {
+
+/// The equation of state of the fluid.
+template <size_t ThermoDim = 0, bool IsRelativistic = true>
```

Comment:
Is 0 a valid thermodynamic dimension?  Is there a reason that these template parameters need to have defaults?

Reply:
0 is definitely not a valid thermodynamic dimension, I believe @nilsdeppe mentioned the reason for needing defaults here but I can't remember what it was.  It may have only been needed for the possible extension to a common base class.

Reply:
Oh, that was for when we try to do dimension-agnostic base classes. For this PR I would leave out the defaults :)